include/arch/posix/posix_trace.h

```diff
@@ -13,7 +13,7 @@ extern "C" {
 void posix_print_error_and_exit(const char *format, ...);
 void posix_print_warning(const char *format, ...);
 void posix_print_trace(const char *format, ...);
-/**
```

Comment:
Nit/comment: Quite an unrelated change in this commit..

Reply:
IIRC, I changed `**` to `*` because documentation generator was complaining about this after moving `posix_trace.h` to `include/`.

Reply:
aaa, ok good to know